lage.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  pipeline: {
+    build: ["^build"],
+    lint: ["build"],
+    'lint:fix': [],
+    test: ["build"],
+    format: [],
+    'format:verify': [],
+    clean: [],
+  },
```

Comment:
This syntax may seem a bit cryptic, but it allows saying that one task is dependent on another, or that a task must be completed in topological order. This allows fast execution, and parallelization of multiple projects. Not really a problem here yet, but the ability to only lint packages you touch is very nice.